modules/platforms/dotnet/Apache.Ignite.Core/Properties/SharedAssemblyInfo.cs

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Can we place this file right in `modules/platforms/dotnet` directory? Seems weird that a shared file is inside one of the project dirs.